radixdlt-core/radixdlt/src/main/java/com/radixdlt/tree/PMTAcc.java

```diff
@@ -72,6 +72,7 @@
 
   private List<PMTNode> visitedAcc = new ArrayList<>();
```

Comment:
this should be renamed to outdated or similar and trigger removal after add

Reply:
I believe this isn't related to the current task